index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <section>
+        <header>
+           <div class="logo">
+            <span>My Logo</a></span>
+           </div>
+           <div class="hamburger">
+            <div class="bar"></div>
+            
+           </div> 
+           <nav>
+            <ul class="nav-list">
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+         </nav>
+       </header>
+    </section>
+    <main>
+        <section class="img-bg">
+        <div class="container">
+            <div class="content-wrapper">
+            <div class="main-content">
+                <h1>i'm Amaka Konwea Glad to see you! </h1>
+                <p class="paragraph">I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div>
+                <p class="lets-connect">LET'S CONNECT</p>
+            </div>
+            <div>
+                <ul class="social-links">
+                <li><a href="https://twitter.com/lawrah_xo"><img src="images/twitter.jpg" alt="twitter"></a></li>
+                <li><a href="https://www.linkedin.com/in/laura-konwea-99647a23a/"><img src="images/Linkedin icon.jpg" alt="linkedin"></a></li>
+                <li><a href="#"><img src="images/medium.jpg" alt="meduim"></a></li>
+                <li><a href="https://github.com/lawrahkonwea"><img src="images/github.jpg" alt="github"></a></li>
+                <li><a href="#"><img src="images/angellist.jpg" alt="angellist"></a></li>
+                </ul>
+                
+            </div>
+           </div>
+        </div>
+        </section>
```

Comment:
- [ ] the background image styling is not matching the Figma design, kindly fix the background image position, size, and anything needed to match Figma 
- [ ] kindly add the bottom left border-radius as well to match the figma design 

Your Design           | Figma
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/198005197-586485cb-9b94-4f6d-9088-edb24d6174bd.png) | ![image](https://user-images.githubusercontent.com/89282221/198005332-79239dbb-d3b2-4e55-9717-c379e633423b.png)